ui/raidboss/data/07-dt/alliance/jeuno-first-walk.ts

```diff
@@ -0,0 +1,808 @@
+          return output.outAtDirection!({ safeDir: output[dirOutputIndex]!() });
+        }
+        return output.getOut!();
+      },
+      outputStrings: {
+        getOut: Outputs.out,
+        outAtDirection: {
+          en: 'Get out toward ${safeDir}',
+        },
+        dirN: Outputs.north,
+        dirE: Outputs.east,
+        dirS: Outputs.south,
+        dirW: Outputs.west,
+        dirNE: Outputs.dirNE,
+        dirSE: Outputs.dirSE,
+        dirSW: Outputs.dirSW,
+        dirNW: Outputs.dirNW,
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Baleful Breath',
+      type: 'StartsUsing',
+      netRegex: { id: '9BF2', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 7,
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Sharp Spike Collect',
+      type: 'HeadMarker',
+      netRegex: { id: '0156', capture: true },
+      run: (data, matches) => data.spikeTargets.push(matches.target),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Sharp Spike Call',
+      type: 'StartsUsing',
+      netRegex: { id: '9F97', source: 'Fafnir The Forgotten', capture: false },
+      delaySeconds: 0.5,
+      durationSeconds: 6.5,
+      alertText: (data, _matches, output) => {
+        if (data.spikeTargets?.includes(data.me))
+          return output.cleaveOnYou!();
+        return output.avoidCleave!();
+      },
+      run: (data) => {
+        // Dragon Breath is also deleted here because Sharp Spike
+        // consistently follows Dragon Breath,
+        // while Touchdown does not.
+        data.spikeTargets = [];
+        delete data.dragonBreathFacingNumber;
+      },
+      outputStrings: {
+        cleaveOnYou: Outputs.tankCleaveOnYou,
+        avoidCleave: Outputs.avoidTankCleaves,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Horrid Roar Spread',
+      type: 'HeadMarker',
+      netRegex: { id: '01F3', capture: true },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Absolute Terror',
+      type: 'StartsUsing',
+      netRegex: { id: '9F8D', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      response: Responses.goSides(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Winged Terror',
+      type: 'StartsUsing',
+      netRegex: { id: '9F8F', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      response: Responses.goMiddle(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Hurricane Wing Raidwide',
+      type: 'StartsUsing',
+      netRegex: { id: '9F71', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 7,
+      infoText: (_data, _matches, output) => output.outerFirst!(),
+      outputStrings: {
+        outerFirst: {
+          en: 'Raidwide x10',
```

Comment:
This should probably be `AoE x10` to be consistent with our other raidwide triggers.